testgrid/conformance/README.md

```diff
@@ -1,43 +1,69 @@
 # Display Conformance Tests with Testgrid
 
-This directory contains tooling for displaying [kubernetes conformance test](https://github.com/cncf/k8s-conformance) results on [testgrid](../README.md).
+This directory contains tooling for uploading [Kubernetes Conformance test](https://github.com/cncf/k8s-conformance) results for display / monitoring on [TestGrid](../README.md).
 
+Federated conformance test results are hosted on the TestGrid [conformance dashboards](https://k8s-testgrid.appspot.com/conformance-all), including the "all" dashboard, and specific sub-dashboards, see [the TestGrid README](../README.md#dashboards) for details on dashboards.  
 
-## Using
+All Kubernetes cluster providers are invited to post results from their conformance test jobs and results from reliable continuous integration against the release branches may even be used as a signal by the Kubernetes release team in the [release-blocking dashboards](https://k8s-testgrid.appspot.com/sig-release-master-blocking).
 
-1. First you will need to set up a publicly readable GCS bucket per [contributing test results](https://github.com/kubernetes/test-infra/blob/master/docs/contributing-test-results.md#contributing-test-results).
+For the original design doc and further details on the motivation please see [design.md](./design.md).
 
-2. Make a PR to test-infra adding your bucket to the testgrid config (again see [contributing test results](https://github.com/kubernetes/test-infra/blob/master/docs/contributing-test-results.md#contributing-test-results)).
+## Usage Guide
 
-3. Run the conformance tests, you can do this by:
+1. First you will need to set up a publicly readable GCS bucket per [contributing test results](https://github.com/kubernetes/test-infra/blob/master/docs/contributing-test-results.md#contributing-test-results) to host your jobs' results.  
+If you cannot or do not want to set up a GCS bucket and only wish to post conformance test results, please contact [@BenTheElder](https://github.com/BenTheElder) or more generally [the gke-kubernetes-engprod team](mailto:gke-kubernetes-engprod@google.com) to arrange for a Google [GKE](https://cloud.google.com/kubernetes-engine/) EngProd provided / maintained bucket for hosting your results. A bucket will be provided following the [playbook](./creating-a-bucket.md) for this.
```

Comment:
What is the motivation for folks to create their own buckets vs always asking to use ours. Is there a limit or constraint if they decide to use ours? Can you briefly talk about that out here plz

Reply:
sure, i meant to cover that, if they create their own they can upload anything they want (other artifacts, other tests entirely) and testgrid / gubernator will display them, if we provide one keep it to conformance test results.

Reply:
added a little more detail